Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM node:lts
+FROM okteto/node:16 as builder
```

Comment:
Do we want to move to Node 18, since it is now the LTS? I think 16 is now considered out-dated.

Reply:
we can do this in another ticket, i dont wanna upgrade to 18 and retest everyhting.